docker/1.3-1/final/Dockerfile.cpu

```diff
@@ -11,6 +11,10 @@ ARG SAGEMAKER_XGBOOST_VERSION
 COPY requirements.txt /requirements.txt
 RUN python3 -m pip install -r /requirements.txt && rm /requirements.txt
 
+# Install smdebug from source
+RUN python3 -m pip install git+https://github.com/awslabs/sagemaker-debugger.git@1.0.29
```

Comment:
Curious, why are we installing from branch here if it's available through pip?

Reply:
so this new version is not available through pip, according to them they haven't released wheels recently and their pipeline for that is blocked. They are not sure when they would be able to resolve that and suggested me to directly install it from source as almost all of their dependencies are installing from source only. To unblock this issue for time being if there are no major cons to this approach maybe we can treat it as a temporary work around and can request them to release a new wheel eventually.

Reply:
Ok, makes sense, thanks for the explanation.